install.sh

```diff
@@ -991,6 +1082,7 @@ configure_collectd() {
 parse_args_wrapper "$@"
 set_variables
 determine_os
+perform_checks
```

Comment:
might want to move before determine_os since this can stop the install and determine_os does not